notebooks/test/executed_notebooks/BigQuery Basics_output.ipynb

```diff
@@ -0,0 +1,615 @@
+       "    <tr>\n",
+       "      <th>6</th>\n",
+       "      <td>Maria</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>7</th>\n",
+       "      <td>Frances</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>8</th>\n",
+       "      <td>Margaret</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>9</th>\n",
+       "      <td>Helen</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "       name\n",
+       "0      Mary\n",
+       "1      Ruby\n",
+       "2     Annie\n",
+       "3    Willie\n",
+       "4      Ruth\n",
+       "5    Gladys\n",
+       "6     Maria\n",
+       "7   Frances\n",
+       "8  Margaret\n",
+       "9     Helen"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "query = \"\"\"\n",
+    "    SELECT name FROM `bigquery-public-data.usa_names.usa_1910_current`\n",
+    "    WHERE state = \"TX\"\n",
+    "    LIMIT 10\n",
+    "\"\"\"\n",
+    "query_job = client.query(\n",
+    "    query,\n",
+    "    # Location must match that of the dataset(s) referenced in the query.\n",
+    "    location=\"US\",\n",
+    ")  # API request - starts the query\n",
+    "\n",
+    "df = query_job.to_dataframe()\n",
+    "df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create a new dataset\n",
+    "\n",
+    "A dataset is contained within a specific [project](https://cloud.google.com/bigquery/docs/projects). Datasets are top-level containers that are used to organize and control access to your [tables](https://cloud.google.com/bigquery/docs/tables) and [views](https://cloud.google.com/bigquery/docs/views). A table or view must belong to a dataset, so you need to create at least one dataset before [loading data into BigQuery](https://cloud.google.com/bigquery/loading-data-into-bigquery)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import time\n",
+    "\n",
+    "# Define a name for the new dataset.\n",
+    "dataset_id = 'test_dataset_1548268897503'\n",
+    "\n",
+    "# Create a DatasetReference using a chosen dataset ID.\n",
+    "# The project defaults to the Client's project if not specified.\n",
+    "dataset_ref = client.dataset(dataset_id)\n",
+    "\n",
+    "# Construct a full Dataset object to send to the API.\n",
+    "dataset = bigquery.Dataset(dataset_ref)\n",
+    "# Specify the geographic location where the dataset should reside.\n",
+    "dataset.location = 'US'\n",
+    "\n",
+    "# Send the dataset to the API for creation.\n",
+    "dataset = client.create_dataset(dataset)  # API request"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load data from a Pandas DataFrame to a new table"
```

Comment:
Nit. pandas never capitalizes itself in the [pandas docs](https://pandas.pydata.org/pandas-docs/stable/), even at the start of sentences.

Reply:
Updated